config/routes.rb

```diff
@@ -65,6 +65,21 @@ def external_redirect(path_prefix, target)
   scope Whitehall.router_prefix, shallow_path: Whitehall.router_prefix do
     root to: redirect("/", prefix: ""), via: :get, as: :main_root
 
+    # Redirects rendered by Whitehall
```

Comment:
I'm a bit less sure about this grouping - do we anticipate doing the redirects outside of whitehall at some point? I guess I'm thinking that maybe it's still useful to see all the e.g. organisation stuff grouped together, even if it's a redirect, unless we know that we'll fix it at some point, in which case this list serves as a to-do list.

But this not at all a strong opinion, just discussion point.

Reply:
I'd like to remove these redirects from Whitehall completely, as router can handle the concept of redirects.  Maybe I'll do that as part of this card, but on a later PR.